key_gen.py

```diff
@@ -0,0 +1,3 @@
+import random
+key = ''.join([random.SystemRandom().choice('abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)') for i in range(50)])
```

Comment:
This is cool, I like how you set a file to do this secret key generation. It's good reusability and accessibility.